include/options/abstract-option.php

```diff
@@ -240,4 +221,13 @@ protected function sanitize( $value, Options $options ) { // phpcs:ignore Variab
 	 * @phpstan-return array{type: SchemaType}
 	 */
 	abstract protected function get_specific_schema(): array;
```

Comment:
```suggestion
	abstract protected function get_data_structure(): array;
```
So it's a bit more precise IMHO.

Reply:
I like it.

Reply:
Done in [fa60b4f](https://github.com/polylang/polylang/pull/1481/commits/fa60b4f804c1c5aba7a54695a84fbede0fed7670).